### PR TITLE
Extension safe api set to YES

### PR DESCRIPTION
**Pull request by hamchapman**
Seems like this used to be the case: https://github.com/krzyzanowskim/CryptoSwift/issues/164 but now isn't

Is that for a reason?

Checklist:
- [x] Pull request against a `develop` branch.
- [x] Correct file headers (see CONTRIBUTING.md).
- [x] Formatted with [SwiftFormat](https://github.com/nicklockwood/SwiftFormat).
- [x] Tests added. (n/a)

Changes proposed in this pull request:

- Set `APPLICATION_EXTENSION_API_ONLY = YES`

Otherwise you get this warning when you have it set to `YES` in your own target:

screen shot 2018-03-09 at 14 33 56

### CODE REVIEW

File: .swift-version [hunk 1]
```diff
@@ -1 +1 @@
-4.1
\ No newline at end of file
+4.1-DEVELOPMENT-SNAPSHOT-2018-03-08-a
```
Comment (krzyzanowskim) [inline .swift-version]: no. This supposes to be ready for 4.1 release. I know .swift-version is restrictive, but snapshot will outdate like.... in few days.

Comment (hamchapman) [reply .swift-version]: Okay cool - I'll remove that change and close the other PR 👍

Comment (hamchapman) [reply .swift-version]: Updated

**Comment by krzyzanowskim** [comment]
thanks!

**Pull request merged**